### PR TITLE
Parra

### PR DESCRIPTION
**Pull request by aparra-11**

### CODE REVIEW

**Review by GenesisMRL — APPROVED** [review]

File: src/components/ChatBox/ChatBox.tsx [hunk 1]
```diff
@@ -40,6 +40,13 @@ const ChatBox = ({ isOpen }: Props) => {
     setInput("");
   };
 
+  const handleKeyPress = (e: any) => {
+    if (e.key === "Enter") {
+      console.log("caca");
```
Comment (GonVieira) [inline src/components/ChatBox/ChatBox.tsx:45]: Bom console log.

File: src/components/ChatBox/styles.ts [hunk 2]
```diff
@@ -94,8 +93,6 @@ export const AIText = styled.p`
   text-align: flex-start;
   width: 80%;
   color: black;
-  background-color: whitesmoke;//rgba(189, 220, 240, 1);
+  background-color: whitesmoke; //rgba(189, 220, 240, 1);
```
Comment (GonVieira) [inline src/components/ChatBox/styles.ts:96]: Tira os comments.

File: src/components/Sidebar/Sidebar.tsx [hunk 3]
```diff
@@ -164,6 +204,7 @@ const Sidebar: any = () => {
               Your Cart
             </h2>
             <ProductsContainer>
+              {console.log(context)}
```
Comment (GonVieira) [inline src/components/Sidebar/Sidebar.tsx:207]: Logs.

File: src/components/navbar/Navbar.tsx [hunk 4]
```diff
@@ -21,9 +21,22 @@ const Navbar = () => {
...
 
   function handleClick() {
-
-    dispatch({ type: "SEND_OPENED", payload: { opened:true } })
+    dispatch({ type: "SEND_OPENED", payload: { opened: true } });
   }
+
+  useEffect(() => {
+    const cartRecovery = localStorage.getItem("shoppingCart");
+    console.log(cartRecovery);
```
Comment (GonVieira) [inline src/components/navbar/Navbar.tsx:29]: Logs

**Review by GonVieira — APPROVED** [review]

**Pull request merged**